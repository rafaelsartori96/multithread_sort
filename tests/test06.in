83458, 41726, 88213, 71762, 70401, 2914, 10967, 81872, 82331, 67187, 77013, 29545
