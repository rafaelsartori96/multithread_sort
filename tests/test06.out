2914 10967 29545 41726 67187 70401 71762 77013 81872 82331 83458 88213
